src/docs/rialto-servermanager-api.json

```diff
@@ -0,0 +1,314 @@
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "registerRequest": {
+      "type": "object",
+      "properties": {
+        "event": {
+          "$ref": "#/definitions/eventType"
+        },
+        "id": {
+          "$ref": "#/definitions/eventId"
+        }
+      },
+      "required": [
+        "event",
+        "id"
+      ]
+    },
+    "unregisterRequest": {
+      "type": "object",
+      "properties": {
+        "event": {
+          "$ref": "#/definitions/eventType"
+        },
+        "id": {
+          "$ref": "#/definitions/eventId"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "activateResult": {
+      "type": "object",
+      "default": null,
+      "description": "",
+      "properties": {
+        "socket": {
+          "$ref": "#/definitions/socket"
+        }
+      },
+      "required": [
+        "socket"
+      ]
+    },
+    "stateResult": {
+      "type": "object",
+      "default": null,
+      "description": "",
+      "properties": {
+        "state": {
+          "$ref": "#/definitions/state"
+        }
+      },
+      "required": [
+        "state"
+      ]
+    },
+    "getListenersResult": {
+      "type": "object",
+      "properties": {
+        "listeners": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/listener"
+          }
+        }
+      },
+      "required": [
+        "listeners"
+      ]
+    }
+  },
+  "methods": {
+    "activate": {
+      "summary": "Activate an app session and spawn a socket",
+      "events": [
+        "StateChangeEvent"
+      ],
+      "params": {
+        "$ref": "#/definitions/activateRequest"
+      },
+      "result": {
+        "$ref": "#/definitions/activateResult"
+      },
+      "errors": [
+      ]
+    },
+    "stop": {
+      "summary": "Stops an app session and closes the socket",
+      "events": [
+        "StateChangeEvent"
+      ],
+      "params": {
+        "$ref": "#/definitions/stopRequest"
+      },
+      "result": {
+        "$ref": "#/common/results/void"
+      },
+      "errors": [
+      ]
+    },
+    "deactivate": {
+      "summary": "Suspends an app session",
+      "events": [
+        "StateChangeEvent"
+      ],
+      "params": {
+        "$ref": "#/definitions/deactivateRequest"
+      },
+      "result": {
+        "$ref": "#/common/results/void"
+      },
+      "errors": [
+      ]
+    },
+    "getState": {
+      "summary": "Gets session state",
+      "params": {
+        "$ref": "#/definitions/getStateRequest"
+      },
+      "result": {
+        "$ref": "#/definitions/stateResult"
+      },
+      "errors": [
+      ]
+    },
+    "register": {
+      "summary": "Registers event listener",
+      "params": {
+        "$ref": "#/definitions/registerRequest"
+      },
+      "result": {
+        "$ref": "#/common/results/default"
+      },
+      "errors": [
+        {
+          "$ref": "#/common/errors/invalid_params"
```

Comment:
Register throws following error:
 -1, "Only one registration per event type per each connection"
 Unregister:
 -1, "Registration info not found")
 
 So #/common/errors/registration_not_found code shall be set to -1